src/rules/sort-collections.ts

```diff
@@ -137,12 +140,15 @@ export const rule = createRule<Options>({
 		docs: {
 			category: "Best Practices",
 			description:
-				"Dependencies, scripts, and configuration values must be declared in alphabetical order.",
+				"Selected package.json collections must be in a consistent order (lexical for most; lifecycle-aware for scripts).",
 			recommended: true,
 		},
 		fixable: "code",
 		messages: {
-			notAlphabetized: "Package {{ key }} are not alphabetized",
+			incorrectOrderLexical:
+				"Entries in '{{ key }}' are not lexically sorted",
```

Comment:
Similar to the above, curious about the use of "lexically" instead of "lexicographically"?